src/main/java/lotto/LottoMain.java

```diff
@@ -0,0 +1,30 @@
+package lotto;
+
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import static lotto.LottoGenerator.generateLotto;
+import static lotto.LottoStatistics.calculateStatistics;
+import static lotto.LottoStatistics.calculateYield;
+
+public class LottoMain {
+
+    public static void main(String[] args) {
+        // 로또 구매
+        BigDecimal amount = InputView.inputPurchaseAmount();
+        List<Lotto> lotto = generateLotto(amount);
+        OutputView.printLotto(lotto);
+
+        // 로또 당첨 번호 입력
+        Lotto winningLotto = InputView.inputWinningNumbers();
+
+        // 당첨 통계 및 수익률 출력
+        Map<LottoRank, Integer> statistics = calculateStatistics(lotto, winningLotto);
```

Comment:
```suggestion
        LottoStatistics statistics = calculateStatistics(lotto, winningLotto);
```
Map을 포장한 일급 콜렉션이 있는데 이 객체를 적극적으로 사용해 보면 어떨까?

Reply:
피드백 감사합니다! 말씀해주신 대로 `LottoStatistics` 타입으로 받도록 수정하려고 보니 기존에는 `static` 메서드를 사용해서 `Map`을 파라미터로 넘기는 방식이었는데, `LottoStatistics`를 쓰려면 결국 내부 Map을 다시 꺼내야 하더라고요.. 찾아보니 `record`는 자동으로 `public getter`를 만들어서 `Map`이 외부에 노출되는 문제가 있다는 것을 알게 되었습니다.. 그래서 `Map.copyOf()`로 불변화하고, 인스턴스 메서드로 바꿔서 `statistics.calculateYield(amount)` 형태로 객체에게 직접 일을 시키도록 변경했습니다. 일급 컬렉션을 왜 강조하시는지 알 것 같아요.. 감사합니다! 🥹